index.html

```diff
@@ -85,9 +89,12 @@ <h1>
                   </div>
                 </div>
                 <div class="hero_button-price">
-                    <a class="selfdecode_action_box " href="https://upacjenta.pl/domowe-badania-krwi"
-                      ><p>order now</p><i class="fa-solid fa-circle-chevron-right"></i></a
-                    >
+                  <a
```

Comment:
close this in div and name it action_box
remove <p> stay alone with text inside a
name <a class="action_link"